src/Controllers.php

```diff
@@ -33,6 +33,13 @@ public function connect(Silex\Application $app)
             ->value('page', '')
             ->assert('page', '.*');
 
+        $ctr->get('/sitemap/{version}.xml', [$this, 'sitemap'])
+            ->bind('sitemap')
+            ->value('version', '');
```

Comment:
`/sitemap/.xml`? I don't think you can or need to set version's default value to empty string here.